modules/wallet/outputs.go

```diff
@@ -90,7 +93,8 @@ func (w *Wallet) Balance(full bool) (total consensus.Currency) {
 			if !full && knownOutput.age > w.age-AgeDelay {
 				continue
 			}
-			total += knownOutput.output.Value
+			// TODO: ???
+			total.Add(knownOutput.output.Value)
```

Comment:
oh. yeah it probably is, because the wallet is getting its inputs from the user. So the wallet should check for that and return an error if it overflows.

The wallet has a few protections but I still think you could sneak an overflow through somewhere.